lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
I think everything under config/app is already in the autoload paths. Certainly app/models.